### PR TITLE
Fix test service

### PR DESCRIPTION
**Pull request by VLiamin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/905

### CODE REVIEW

File: Tests.HealthCheck/Models/Helpers/SmtpGetter.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            _rcGetSmtp = rcGetSmtp;
+        }
+
+        public async Task<bool> GetSmtp()
+        {
+            try
+            {
+                IGetSmtpCredentialsResponse response = (await _rcGetSmtp.GetResponse<IOperationResult<IGetSmtpCredentialsResponse>>(
+                  IGetSmtpCredentialsRequest.CreateObj())).Message.Body;
```
Comment (BelehovEgor) [inline Tests.HealthCheck/Models/Helpers/SmtpGetter.cs]: mb you should check the success of request?

File: Tests.HealthCheck/Startup.cs [hunk 2]
```diff
@@ -128,6 +135,22 @@ public void ConfigureServices(IServiceCollection services)
...
                 })
                 .AddInMemoryStorage();
+
+            services.AddMassTransit(x =>
+            {
+                x.UsingRabbitMq((context, cfg) =>
+                {
+                    cfg.Host("localhost", "/", host =>
+                    {
+                        host.Username("TestService_971E75B1-E475-4A2D-97A4-9A7FDE1FK8R9");
```
Comment (BelehovEgor) [inline Tests.HealthCheck/Startup.cs]: add getting it from env and appsettings

**Review by Cheephot — APPROVED** [review]

File: Tests.HealthCheck/Models/Helpers/SmtpGetter.cs (hunk 1 above)
Comment (VLiamin) [reply Tests.HealthCheck/Models/Helpers/SmtpGetter.cs]: done

File: Tests.HealthCheck/Startup.cs (hunk 2 above)
Comment (VLiamin) [reply Tests.HealthCheck/Startup.cs]: done

**Review by MariyaSin — APPROVED** [review]

File: Tests.HealthCheck/ReportEmailSender.cs [hunk 3]
```diff
@@ -26,50 +24,43 @@ public static void AddReport(UIHealthReport newReport)
         public static void Start(int interval, string[] emails)
         {
             _emails = emails.ToList();
-            _interval = TimeSpan.FromMinutes(interval);
+
+            _emails[0] = "spartak.ryabtsev@lanit-tercom.com";
```
Comment (BelehovEgor) [inline Tests.HealthCheck/ReportEmailSender.cs]: i guess we have to get this email from env or appsettings

Comment (VLiamin) [reply Tests.HealthCheck/ReportEmailSender.cs]: done

**Review by BelehovEgor — APPROVED** [review]

**Pull request merged**